crates/delta/src/routes/servers/server_edit.rs

```diff
@@ -57,6 +58,11 @@ pub async fn edit(
         permissions.throw_if_lacking_channel_permission(ChannelPermission::ManageServer)?;
     }
 
+    // Check we are the server owner if changing sensitive fields
+    if data.owner.is_some() && user.id != server.owner {
```

Comment:
This should probably also be bypassed by `user.privileged` (?)

Reply:
If you check the mongodb database and the relevant collection for users, I think there is a variable named "privileged" however I could be wrong

Reply:
> If you check the mongodb database and the relevant collection for users, I think there is a variable named "privileged" on a user object however I could be wrong

nah, it has this info
```json
  {
    _id: '01JNBQ74E52A4ZPY58QSANM6X7',
    username: 'lazycat',
    discriminator: '4348',
    relations: []
  },
```

Reply:
now privileged users can change owner of any server